routes/anonymousDeck.js

```diff
@@ -52,9 +56,10 @@ router.post("/create", async function (req, res) {
       }
     }
 
-    let deck = Object(req.body);
-    deck.name = String(deck.name || "");
-    deck.profiles = Object(deck.profiles);
+    let deck = Object();
+    deck.editing = Boolean(req.body.editing);
```

Comment:
I'm a bit confused - when is the /create req.body.editing used and when is /edit used?

Reply:
I think there is some useless code that I left in, and I forgot to remove some custom /create profile request functions and I think one for edit. Instead, it's all included in the ones that existed previously now. I think there are some request functions that can be deleted as they aren't used.